### PR TITLE
sparsity+src

### PR DESCRIPTION
**Pull request by vincentliuk**
sparsity code with several src codes

### CODE REVIEW

**Comment by vincentliuk** [comment]
@subutai  Here are the python codes for sparsity and several other source codes. I have tested it on jupyter notebooks. However, there might be some data format difference which could be solved manually.

**Comment by subutai** [comment]
@lscheinkman Can you take a look? Thanks.

File: projects/high_order_cell_assembly/get_response_mat.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+def get_response_mat(spiketrain, imgPara, stimType, goodCells, plotRaster):
```
Comment (lscheinkman) [inline projects/high_order_cell_assembly/get_response_mat.py]: Need to import  `numpy`

File: projects/high_order_cell_assembly/sparsity_over_trial.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import numpy as np
+import scipy.io as sio 
+import matplotlib.pyplot as plt
+import get_response_mat
+import calculate_sparsity_over_trials
```
Comment (lscheinkman) [inline projects/high_order_cell_assembly/sparsity_over_trial.py]: Replace
```
import get_response_mat
import calculate_sparsity_over_trials
```
With
``` 
from get_response_mat import get_response_mat
from calculate_sparsity_over_trials import calculate_sparsity_over_trials
```

**Comment by lscheinkman** [comment]
Calling `python sparsity_over_trial.py` from command line does not work. Here is the error:
```
Traceback (most recent call last):
  File "sparsity_over_trial.py", line 24, in <module>
    stim_type = imgPara['stim_type']
IndexError: only integers, slices (`:`), ellipsis (`...`), numpy.newaxis (`None`) and integer or boolean arrays are valid indices
```

Can you add the notebook you used to test?

**Comment by lscheinkman** [comment]
Replaced by #838

**Pull request closed without merging**